taskcluster/kinds/beetmover/kind.yml

```diff
@@ -21,145 +21,73 @@ tasks:
     android-arm64:
         requires-level: 3
         worker-type: beetmover
-        worker:
-            chain-of-trust: true
-            max-run-time: 1800
         run-on-tasks-for: [action]
```

Comment:
It needs to at least be `run-on-tasks-for: []` because the default is to run everywhere. I left `action` in because I figured it could act as an indicator that these are meant to be triggered via action to those who are unfamiliar with them.